src/lib.rs

```diff
@@ -87,21 +97,42 @@ fn parse_sealed(item: syn::Item) -> syn::Result<proc_macro2::TokenStream> {
 }
 
 // Care for https://gist.github.com/Koxiaet/8c05ebd4e0e9347eb05f265dfb7252e1#procedural-macros-support-renaming-the-crate
-fn parse_sealed_struct(strct: syn::ItemStruct) -> syn::Result<proc_macro2::TokenStream> {
-    let ident = &strct.ident;
+fn parse_sealed_trait(mut item_trait: syn::ItemTrait) -> syn::Result<proc_macro2::TokenStream> {
+    let trait_ident = &item_trait.ident;
+    let trait_generics = &item_trait.generics;
```

Comment:
```suggestion
    let trait_generics = item_trait.generics.split_for_impl().1;
```

Basically if you had `trait Foo<T : Send>`, you want to add `Sealed<T>` as a supertrait, not `Sealed<T : Send>`. The handy `.split_for_impl()` yields a triplet with:
 1. the "into generics" (`T : Send`),
 2. the "forwarding generics" (`T`) [we want this here],
 3. and the `where` clauses, if any